### PR TITLE
Reduce calls to Patch advisories to a single request

### PR DESCRIPTION
**Pull request by PaulWay**
The subtotals for the three advisory types - bugs, security and enhancements - are available in the `meta` structure of the request. This means that we do not have to make three requests to the Patch API in order to get the data necessary to build the Advisory Types pie chart.

This work basically generates a single request and stores the subtotals data.  The subtotal for each advisory type are then pulled out in the PatchManagerCard component.

Signed-off-by: Paul Wayper <paulway@redhat.com>

### CODE REVIEW

File: src/SmartComponents/PatchManager/PatchManagerCard.js
Comment (mkholjuraev) [inline src/SmartComponents/PatchManager/PatchManagerCard.js:34]: As @gkarat mentioned, we need to handle the initial render when `advisories` is undefined. We can achieve this with object destruction: 
```suggestion
    const { security, bugfix: bugs, enhancement: enhancements } = advisories || {};
```

File: src/SmartComponents/PatchManager/PatchManagerCard.js
Comment (gkarat) [inline src/SmartComponents/PatchManager/PatchManagerCard.js:36]: `advisories` are undefined before the network request is fulfilled. This leads to an error that breaks the whole page component because you are accessing `security` and further parameters on undefined type.

The very first thing came to mind and that fixes this is to provide an alternative value: 
```suggestion
    const security = advisories?.security || 0;
    const bugs = advisories?.bugfix || 0;
    const enhancements = advisories?.enhancement || 0;
```

**Review by gkarat — CHANGES_REQUESTED** [review]
Hi! Thanks for your PR! when deployed locally with this PR, the page crashed. but I found the cause of the problem and described it below.

but overall LGTM, it's a good improvement

**Review by mkholjuraev — APPROVED** [review]
LGTM! Works as expected. @PaulWay Thank you.

**Review by mkholjuraev — APPROVED** [review]

**Comment by PaulWay** [comment]
How is this looking - does anything else need to be done before it can be merged?

File: src/SmartComponents/PatchManager/PatchManagerCard.js
Comment (PaulWay) [reply src/SmartComponents/PatchManager/PatchManagerCard.js]: I think this is fixed with `const { security, bugfix: bugs, enhancement: enhancements } = advisories || {};` now?

**Comment by mkholjuraev** [comment]
I have approved this PR, but there is one pending change request by @gkarat .

**Pull request merged**
